src/scenes/ProgressReports/Detail/ProgressReportDrawer/ProgressReportSidebar.tsx

```diff
@@ -1,20 +1,27 @@
 import { InfoOutlined } from '@mui/icons-material';
 import { Box, Button, Typography } from '@mui/material';
 import { useDialog } from '~/components/Dialog';
+import { useProgressReportContext } from '../../ProgressReportContext';
 import { InstructionsDialog } from './InstructionsDialog';
 import { ProgressReportStepper } from './ProgressReportStepper';
 
-export const ProgressReportSidebar = ({ step }: { step: number }) => {
+export const ProgressReportSidebar = () => {
+  const { step, currentReport } = useProgressReportContext();
   const [instructionsState, showInstructions] = useDialog();
 
+  const daysLeft = Math.floor(currentReport?.due.diffNow('days').days ?? 0);
+
   return (
     <Box
       sx={{
         display: 'initial',
         flexDirection: 'column',
-        minWidth: 350,
+        width: 300,
         padding: 2,
         pt: 4,
+        right: 0,
+        top: 0,
+        position: 'fixed',
```

Comment:
This has me 😬 a little bit. Esp the part where the main content separately has to declare `mr: 300px`. Could we use flexbox or grid here instead?

Reply:
Not really because of the position fixed :( unfortunately that breaks the standard grids. If you get something working, I'm happy to change it, but from what I've tried, the built-in solutions didn't work. Also the main container does not declare `mr: 300px`, it declares `width: 'calc(100% - 300px)'`